lib/rules/no-jquery.js

```diff
@@ -27,62 +28,93 @@ module.exports = {
   ERROR_MESSAGE,
 
   create(context) {
-    let destructuredAssignment;
-    let emberImportAliasName;
-    let jqueryImportAliasName;
     const report = function (node) {
       context.report(node, ERROR_MESSAGE);
     };
 
+    /**
+     * Reports each reference to the variable
+     *
+     * @param  {Variable} scopeVariable variable to report references for
+     * @return {number}                 the number of references to the variable
+     */
+    function reportVariableReferences(scopeVariable) {
+      // report each reference to the variable
+      scopeVariable.references.forEach((reference) => {
+        report(reference.identifier.parent);
+      });
+
+      return scopeVariable.references.length;
+    }
+
     return {
-      ImportDeclaration(node) {
-        emberImportAliasName = ember.getEmberImportAliasName(node);
+      /**
+       * Report all references whe importing jQuery
```

Comment:
I would prefer to maintain the existing behavior of the rule of only flagging the function calls and not the import statements since this ends up being a duplicate violation.

Reply:
I managed to set it up so that we report any references to an imported value, then once all the violations have been fixed, we can report the import statement

While there are references to the imported value, it shouldn't report line with the import statement

I think that should avoid the duplicate violation issue, what do you think?

Reply:
I don't think we need to worry about reporting the import statement since if the import statement is unused, it will get flagged by the recommended eslint rule [no-unused-vars](https://eslint.org/docs/rules/no-unused-vars) anyway.

Reply:
Right, yes.. good point! 😅